debian/patches/ocf-suexec.patch

```diff
@@ -32,8 +35,11 @@
 +    if (((uid != dir_info.st_uid) ||
 +         (gid != dir_info.st_gid) ||
 +         (uid != prg_info.st_uid) ||
-+         (gid != prg_info.st_gid)
-+        ) && !(dir_info.st_uid == 0 && dir_info.st_gid == 0 && prg_info.st_uid == 0 && prg_info.st_gid == 0)) {
++         (gid != prg_info.st_gid)) &&
++        !((dir_info.st_uid == 0) &&
++          (dir_info.st_gid == 0) &&
++          (prg_info.st_uid == 0) &&
++          (prg_info.st_gid == 0))) {
```

Comment:
did anything actually change here? i don't think it did but reading patches of patches is always hard :)

Reply:
Nah, nothing actually changed here besides formatting, so this part should behave the same.